apps/react-nextjs/.env

```diff
@@ -1 +1,4 @@
-NEXT_PUBLIC_ORG_ID=
\ No newline at end of file
+# internal_testing_2
+NEXT_PUBLIC_ORG_ID=00000000-0000-0000-0200-000000000000
```

Comment:
We talked to @balenio about this and agreed we should use the internal testing orgs since we already have a cleanup script that removes old suborgs from the testing environment.

Once we start writing tests that mutate persistent state (eg writing to attributes), each test run will create a separate suborg in order to ensure clean state.